tests/unit/utils/test_objectives.py

```diff
@@ -65,21 +60,33 @@ def test_objective_maps_minimizers_to_minimum(
 
 
 def test_branin_no_function_values_are_less_than_global_minimum() -> None:
-    npt.assert_array_less(tf.squeeze(BRANIN_MINIMUM) - 1e-6, branin(_unit_grid_2d()))
+    samples = Box([0.0, 0.0], [1.0, 1.0]).sample(1000)
```

Comment:
`tf.random.uniform` is more powerful, and would be consistent with similar random sampling across other tests in trieste